Entities/Characters/Archer/ArcherLogic.as

```diff
@@ -342,7 +343,12 @@ void ManageBow(CBlob@ this, ArcherInfo@ archer, RunnerMoveVars@ moveVars)
 		if (hasarrow != this.get_bool("has_arrow"))
 		{
 			this.set_bool("has_arrow", hasarrow);
-			this.Sync("has_arrow", isServer());
+			CBitStream params;
+			params.write_bool(hasarrow);
+			if (isServer())
+				this.SendCommand(this.getCommandID("has arrows"), params);
+			else
+				this.SendCommandOnlyServer(this.getCommandID("has arrows"), params);
```

Comment:
I haven't tested this but couldn't these lines be simplified to something like this?
```angelscript
this.SendCommand(this.getCommandID("has arrows"), params, isServer());
```

Reply:
only CRules SendCommand has this bool at the end, CBlob function doesnt have it